### PR TITLE
added triage guidelines

### PR DESCRIPTION
**Pull request by ghuntley**
## PR Type
What kind of change does this PR introduce?

- Documentation content changes

## What is the new behavior?

Howdy folks, this PR contains my 2cents on how we could (should?) standardize our approach to triaging issues on Github so that there is consistency and to document our expectations. 

Let's discuss in-line on the markdown. ✏️ 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

This document was based off the guidelines authored by the Kubernetes community sig:
https://github.com/kubernetes/community/blob/e4eabb7c3c26ce4463f1ab89e0ad94a7a7671b08/contributors/guide/issue-triage.md

### CODE REVIEW

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+In all cases, if you do not get a response in 20 days then close the issue with an appropriate comment.
+
+## Define priority
+
+We use GitHub issue labels for prioritization. The absence of a priority label means the bug has not been reviewed and prioritized yet.
+
+We try to apply these priority labels consistently across the entire project, but if you notice an issue that you believe to be incorrectly prioritized, please do let us know and we will evaluate your counter-proposal.
+
+If you want an issue resolved faster, please tell the maintainers that you are willing to send in a pull-request and we'll do everything we can to bring you to speed with becoming a contributor.
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: 🖊 to bring you _UP_ to speed with becoming a contributor.

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+Community support requests should be directed to:
+
+* Our [documentation](https://platform.uno/docs/) and [troubleshooting guide](https://platform.uno/docs/troubleshooting-guide).
+* [Stack Overflow](https://stackoverflow.com/questions/tagged/uno-platform).
+* The [Uno gitter room](https://gitter.im/uno-platform/Lobby)
+* On [Twitter using the #unoplatform](https://twitter.com/search?q=%23unoplatform) hashtag.
+
+Organizations that want a deeper level of support beyond community support, should be directed to [contact nventive](https://platform.uno/contact/) to discuss obaining professional support.
+
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: 🖊 _obtaining_

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+## Validate if the issue is a bug
+
+Confirm if the problem is a bug by reproducing it. If a test case has not been supplied ask the reporter to supply one. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
```
Comment (jeromelaban) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]:
```suggestion
Confirm if the problem is a bug by reproducing it. If a test case has not been supplied ask the reporter to supply one, in the the form of an isolated solution built from the Visual Studio project templates. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
```

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+* `priority/important-longterm`: Important over the long term, but may not be currently staffed and/or may require multiple releases to complete.
+
+* `priority/backlog`: There appears to be general agreement that this would be good to have, but we may not have anyone available to work on it right now or in the immediate future. Community contributions would be most welcome in the meantime (although it might take a while to get them reviewed if reviewers are fully occupied with higher priority issues, for example immediately before a release).
+
+* `priority/awaiting-more-evidence`: Possibly useful, but not yet enough support to actually get it done. These are mostly place-holders for potentially good ideas, so that they don't get completely forgotten, and can be referenced or deduped every time they come up.
+
+## Closing issues
+
+Issues that are identified as a support request, duplicate, not-reproducible or lacks enough information from reporter should be closed following guidelines explained in this file. Also, any issues that can not be resolved because of any particular reason should be closed. These issues should have one or more of following self-readable labels:
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: should be closed _using the_ -following- guidelines

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+  The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test**
+
+  There should be existing tests that can be modified, or existing test cases fit to be copied. If the area of code doesn't have tests, before labelling the issue, add a test fixture. This prep often makes a great `help wanted` task!
+
+## Quick Replies
+
+When commenting on an issue or pull request, there's a [feature in GitHub](https://help.github.com/en/articles/using-saved-replies) where you can add a saved reply that you've already set up. The saved reply can be the entire comment or if you want to customize it, you can add or delete content. Below you'll find some guidelines that maintainers and contributors can use as a starting point.
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: <del>guidelines</del> -> suggestions

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+## Define priority
+
+We use GitHub issue labels for prioritization. The absence of a priority label means the bug has not been reviewed and prioritized yet.
+
+We try to apply these priority labels consistently across the entire project, but if you notice an issue that you believe to be incorrectly prioritized, please do let us know and we will evaluate your counter-proposal.
+
+If you want an issue resolved faster, please tell the maintainers that you are willing to send in a pull-request and we'll do everything we can to bring you to speed with becoming a contributor.
+
+* `priority/critical-urgent`: Must be actively worked on as someone's top priority right now. Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. Team leaders are responsible for making sure that all the issues, labeled with this priority, are being actively worked on. Examples include user-visible bugs in core features, broken builds or tests and critical security issues.
```
Comment (davidjohnoliver) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: > user-visible bugs in core features 

I'm not sure what this means. I don't know which ones are the 'core features' and (nearly) all bugs are user-visible. I think the other two criteria (broken builds, security vulnerabilities) are valid.

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+* `triage/duplicate`: Indicates an issue is a duplicate of other open issue.
+* `triage/not-reproducible`: Indicates an issue can not be reproduced as described.
+* `triage/needs-information`: Indicates an issue needs more information in order to work on it.
+* `triage/unresolved`: Indicates an issue that can not be resolved.
+
+## Help Wanted issues
+
+We use two labels [help wanted](https://github.com/nventive/uno/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+sort%3Aupdated-desc) and [good first issue](https://github.com/nventive/uno/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+sort%3Aupdated-desc) to identify issues that have been specially curated for new contributors.
+
+The `good first issue` label is a subset of `help wanted` label, indicating that members have committed to providing extra assistance for new contributors. All good `first issue` items also have the `help wanted` label.
```
Comment (davidjohnoliver) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]:
```suggestion
The `good first issue` label is a subset of `help wanted` label, indicating that members have committed to providing extra assistance for new contributors. All `good first issue` items also have the `help wanted` label.
```

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+
+Items marked with the `good first issue` label are intended for _first-time
+contributors_. It indicates that members will keep an eye out for these pull
+requests and shepherd it through our processes.
```
Comment (davidjohnoliver) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]:
```suggestion
requests and shepherd them through our processes.
```

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+title: Guidelines for creating and triaging issues
+---
+
+## Purpose
+
+Speed up issue management.
+
+The detailed list of labels can be found at https://github.com/nventive/uno/labels.
+
+While working on triaging issues you may not have privilege to assign a specific label and in that case add a comment in the issue with your findings.
```
[the remark this answers is not in the document]
Comment (ghuntley) [reply doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: There's no such thing as restricted labels but anyone in the community can do issue triage which means they might not have permissions to apply labels. GitHub recently added a `Triage` permission level which we could grant trusted members in the community or the project in the future could head down the k8/chatops route of using commands similar to `/azp run` to label issues.

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md (hunk 6 above)
Comment (ghuntley) [reply doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: This document was based off the guidelines authored by the Kubernetes community sig:
https://github.com/kubernetes/community/blob/e4eabb7c3c26ce4463f1ab89e0ad94a7a7671b08/contributors/guide/issue-triage.md and `core features` comes from there. It has been changed to `time-sensitive hotfixes`

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
+---
+title: Guidelines for creating and triaging issues
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: change name to "guidelines for issues and pull-requests"

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+
+## Validate if the issue is a bug
+
+Confirm if the problem is a bug by reproducing it. If a test case has not been supplied ask the reporter to supply one. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
+
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate. Label such issues as `triage/duplicate`.
+* If you can not reproduce the issue, label it as a `triage/not-reproducible`. Contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If you need more information to further work on the issue, let the reporter know it by adding an issue comment followed by label `triage/needs-information`.
+
+In all cases, if you do not get a response in 20 days then close the issue with an appropriate comment.
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: add information about `triage/needs-information` being automatically closed by a bot. https://probot.github.io/apps/request-info/

File: doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+  Link to examples of similar implementations so new contributors have a reference guide for their changes.
+
+- **Identifies Relevant Code**
+
+  The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test**
+
+  There should be existing tests that can be modified, or existing test cases fit to be copied. If the area of code doesn't have tests, before labelling the issue, add a test fixture. This prep often makes a great `help wanted` task!
+
```
Comment (ghuntley) [inline doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md]: add section about recognising contributors via https://allcontributors.org

**Pull request merged**